source/api/filter.md

```diff
@@ -97,7 +97,18 @@ Executed before a post is rendered. Refer to [post rendering](posts.html#Render)
 For example, to transform the title to lower case:
 
 ``` js
-hexo.extend.filter.register('before_post_render', function(data){
+hexo.extend.filter.register('before_post_render', function(data) {
+  data.title = data.title.toLowerCase();
+  return data;
+});
+```
+
+This filter matches every files in the `source/` folder, it may be necessary to skip non-markdown files, unless those files are listed in [`config.skip_render`](/docs/configuration#Directory); `themes/<your-theme>/source/` folder is not processed by this filter.
```

Comment:
Last time I checked, `before_post_render` only processes files under `source/_posts` directories.

Reply:
I did a check before this PR, both `before_post_render` and `after_post_render` render everything in `source/`.

Reply:
can confirm issue https://repl.it/repls/DefenselessHightechFormats#index.js